dialogue-core/src/main/metrics/dialogue-core-metrics.yml

```diff
@@ -9,6 +9,10 @@ namespaces:
         type: timer
         tags: [service-name]
         docs: Request time, note that this does not include time spent reading the response body.
+      requests.active:
+        type: counter
+        tags: [stage]
```

Comment:
I considered adding `service-name` here as well, that way we have more insight into precisely where our active requests are going. Thoughts?

Reply:
Yes I think adding service name would be good